perf/types_libpfm.go

```diff
@@ -29,12 +29,15 @@ const (
 	perfAttrBitsExcludeGuest = 1 << 20
 )
 
-// ReadFormat allows to read perf event's value for non-grouped events
-type ReadFormat struct {
-	Value       uint64 /* The value of the event */
+type GroupReadFormat struct {
```

Comment:
Provide more documentation, please (at list a doc string similar to the one you deleted).